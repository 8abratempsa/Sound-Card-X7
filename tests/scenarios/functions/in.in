octosql "SELECT 4 IN (1,2,3,4,5), 4 IN (1,2,3,5),
         4 NOT IN (1,2,3,4,5), 4 NOT IN (1,2,3,5)"
