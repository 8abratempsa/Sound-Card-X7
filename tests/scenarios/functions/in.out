+-------+-------+-------+-------+
| col_0 | col_1 | col_2 | col_3 |
+-------+-------+-------+-------+
| true  | false | false | true  |
+-------+-------+-------+-------+
